subsys/net/lib/lwm2m/Kconfig

```
@@ -480,6 +480,21 @@ config LWM2M_RW_SENML_CBOR_RECORDS
 	  The CBOR library requires you to set an upper limit for the records when encoder
 	  and decoder do get generated.
 
+config LWM2M_RESOURCE_DATA_CACHE_SUPPORT
+	bool "Resource Time series data cache support"
+	depends on (LWM2M_RW_SENML_JSON_SUPPORT || LWM2M_RW_SENML_CBOR_SUPPORT)
+	depends on POSIX_CLOCK
+	help
+	  Enable Time series data storage for selected resources. Requires CONFIG_POSIX_CLOCK=y
+	  gettimeofday() to provide current Unix timestamp.
+
+config LWM2M_RESOURCE_DATA_CACHE_ELEMENT_SIZE
+	int "Maxium # of resource count for data cache"
+	default 4
+	range 1 10
+	help
+		This settings define how many different resource client can be enable for data cache.
```

```suggestion
if LWM2M_RESOURCE_DATA_CACHE_SUPPORT

config LWM2M_MAX_CACHED_RESOURCES
	int "Maxium # of cached resources"
	default 4
	range 1 1024
	help
		This settings define how many different resources may have cache enabled.
		This affects static memory usage of engine.

endif # LWM2M_RESOURCE_DATA_CACHE_SUPPORT
```

I don't see the point of range here.

---

Ok I remove range